app/controllers/tickets_controller.rb

```diff
@@ -28,9 +28,20 @@ def create
     end
   end
 
-  # I did this so that the seat_for_screening_is_taken doesnt throw error when you try to edit ticket type or price with
-  # existing screening and seat id
-  # If there is a clearer way to write this feel free to correct me
+  def create_offline
```

Comment:
From my perspecive it would be better to have this action in separated controller. For instance under URL - `POST /offline/tickets` and action `create`. Consider that maybe this approach will be more clean.